be/src/Project/Services/Identity/Identity.Api/Areas/Home/ViewModels/PolicyPageViewModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Identity.Api.Areas.Home.DomainModels;
+using System.Collections.Generic;
+
+namespace Identity.Api.Areas.Home.ViewModels
+{
+    public class PolicyPageViewModel
```

Comment:
**suggestion:** would it be possible to use PrivacyPolicy instead of Policy only everwhere? For consistency